hardhat.config.ts

```diff
@@ -37,6 +46,67 @@ const config: HardhatUserConfig = {
     outDir: 'build/types',
     target: 'ethers-v5',
   },
+  paths: {
+    deploy: './deploy',
+    deployments: './deployments',
+  },
+  namedAccounts: {
+    deployer: {
+      default: 0,
+    },
+  },
+}
+
```

Comment:
Why specify a key and url for each network instead of just `DEPLOYER_KEY` `RPC_URL` and `NETWORK`?

Reply:
good point